project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.7
+sbt.version=1.5.0
```

Comment:
why 1.5.0? / did you try the latest (I think 1.6.2)?

Reply:
I updated to the minimum possible. Because its SBT I would prefer to keep version changes as small as possible in order to reduce risk of changes to our commands - especially while we are already doing a big upgrade.

My general rule is one big upgrade at a time. If we want to upgrade this another major version, we can, but I wouldn't want to do that at the same time as this upgrade in order to keep things as small as possible

Reply:
feels like we really **really** need something like scala steward

Reply:
I agree

Reply:
Did you know `sbt build` isn't a command? I didn't, for some reason I thought that was the command to use - either way, the one sbt command that we use in Sam that is in our readme is `sbt test`, which worked.